napari/layers/labels/labels.py

```diff
@@ -732,15 +710,46 @@ def mode(self):
         Mode.ERASE: 'circle',
     }
 
+    @staticmethod
+    def _mode_help_string(mode: Mode) -> str:
+        """
+        Return string informing about shortcuts
+        that will enable alternative layer mode
+        """
+        from . import _labels_key_bindings as kb
+
+        help_li = []
+        shortcuts = get_settings().shortcuts.shortcuts
```

Comment:
I don't love that we now have this import and the `get_settings` call in here. In general it moves our layer models in the direction of more coupled to the rest of the code base and more complicated - I'd much rather them be moving in the other direction. Can we avoid this pattern here?

Reply:
we could move whole mode string generation outside layers implementation. Then based on the active layer and selected mode, prepare mode string.

Reply:
i think that would be much better - would that also allow us to reduce some of the duplicated code across layer types?